events/api.py

```diff
@@ -1115,7 +1116,7 @@ def get_queryset(self):
         return queryset
 
 
-register_view(KeywordRetrieveViewSet, "keyword")
+register_view(KeywordViewSet, "keyword")
```

Comment:
What was this change?

Reply:
KeywordRetrieveViewSet renamed to more generic KeywordViewSet since the viewset has UpdateModelMixin and DestroyModelMixin.

Reply:
Ah, indeed. :+1: